content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -113,6 +113,14 @@ <h3>Create a new Service</h3>
 				<code><b>echo "NODE_PORT=$NODE_PORT"</b></code></p>
 				<p>Now we can test that the app is exposed outside of the cluster using <code>curl</code>, the IP address of the Node and the externally exposed port:</p>
 				<p><code><b>curl http://"$(minikube ip):$NODE_PORT"</b></code></p>
+        {{< note >}}<p>If Docker is the container runtime, a minikube tunnel is needed. This is because Docker Desktop does not support ports. <br>
```

Comment:
``suggestion
        {{< note >}}<p>If you're running minikube with Docker Desktop as the container driver, a minikube tunnel is needed. This is because containers inside Docker Desktop are isolated from your host computer.<br>
```

What about other minikube drivers, eg `kvm2`?